docs/reference-guides/block-api/block-patterns.md

```diff
@@ -16,6 +16,16 @@ In this Document:
 
 ## Block Patterns
 
+Before using the helper functions listed on this page it is advisable to check for the existence of the `WP_Block_Patterns_Registry` class.
+
+```
+if ( ! class_exists( 'WP_Block_Patterns_Registry' ) ) {
+     return;
+   }
+```
+
+This class only exists in versions of the block editor that support these helper functions.
```

Comment:
Would it be beneficial here to instead say what version of WordPress is required? I.e `This class only exists in WordPress XX and later`